src/components/Search.js

```diff
@@ -0,0 +1,97 @@
+import React, { Fragment } from 'react';
+import { Global, css } from '@emotion/core';
+import { DocSearch } from '@docsearch/react';
+import '@docsearch/react/style';
+
+const Search = () => {
+  return (
+    <Fragment>
+      <Global
+        styles={css`
+          :root {
+            a {
+              border-bottom-style: none;
```

Comment:
You might want to scope this rule under `.DocSearch` instead of `:root` because otherwise it's applied everywhere else, right?

Reply:
Yes indeed. Updated